CHANGELOG.unreleased.md

```diff
@@ -7,10 +7,14 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [VM] Rescan ISO SRs button is also avaible on the console view
+
 ### Bug fixes
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [VM/disks] Fix `an error has occured` when self service user was on VM disk view
```

Comment:
Why?

Reply:
You don't need to add a bug fix entry if the bug hasn't been released.

Reply:
@pdonias The bug is already released. [See xoa-support#3896](https://gitlab.com/vates/xoa-support/-/issues/3896)

Reply:
Right!